### PR TITLE
Fix transliteration error

### PR DESCRIPTION
**Pull request by yshefer**

### CODE REVIEW

File: frontend/src/strings/loc.ts [hunk 1]
```diff
@@ -31,10 +31,11 @@ export function useTransliterate(): (key: string) => string {
 
     return (key: string): string => {
         let result = '';
-
+        let translitedChar;
         for (let i = 0; i < key.length; i++) {
             const char = key[i];
-            result += translit_ru_en[char] || char;
+            translitedChar = translit_ru_en[char];
```
Comment (vladimirshefer) [inline frontend/src/strings/loc.ts:37]: move `let` here

File: frontend/src/strings/loc.ts [hunk 2]
```diff
@@ -44,14 +45,14 @@ export function useTransliterate(): (key: string) => string {
 const translit_ru_en: { [key: string]: string } = {
     'А': 'A', 'а': 'a', 'Б': 'B', 'б': 'b', 'В': 'V', 'в': 'v',
     'Г': 'G', 'г': 'g', 'Д': 'D', 'д': 'd', 'Е': 'E', 'е': 'e',
-    'Ё': 'YO', 'ё': 'yo', 'Ж': 'ZH', 'ж': 'zh', 'З': 'Z', 'з': 'z',
+    'Ё': 'Yo', 'ё': 'yo', 'Ж': 'Zh', 'ж': 'zh', 'З': 'Z', 'з': 'z',
```
Comment (vladimirshefer) [inline frontend/src/strings/loc.ts:48]: this solves the problem with wrong cased letters for
"Яна" -> "Yana"
but introduces the problem for
"ЯНА" -> "YaNA"

Comment (yshefer) [reply frontend/src/strings/loc.ts:48]: Neither YAna nor YaNA looks good. However, names and even nicknames that contain only capital letters are rarer cases, so I'd leave it as is

File: frontend/src/strings/loc.ts (hunk 1 above)
Comment (yshefer) [reply frontend/src/strings/loc.ts:37]: Is it ok to declare a variable inside the loop?

Comment (vladimirshefer) [reply frontend/src/strings/loc.ts:37]: yes

Comment (vladimirshefer) [reply frontend/src/strings/loc.ts:37]: as you can see on line 26 there is a variable declared in the loop
`const char = key[i];`

**Pull request merged**